docs/docs/examples/login/flutter.md

```diff
@@ -97,41 +93,30 @@ To reduce the commented default code, we will modify the `main.dart` file.
 First, the `MyApp` class: it remains a stateless widget:
 
 ```dart reference
-https://github.com/zitadel/zitadel_flutter/blob/main/lib/main.dart#L14-L28
+https://github.com/zitadel/zitadel_flutter/blob/main/lib/main.dart#L45-L79
 ```
 
 Second, the `MyHomePage` class will remain a stateful widget with
 its title, we don't change any code here.
 
 ```dart reference
-https://github.com/zitadel/zitadel_flutter/blob/main/lib/main.dart#L30-L37
+https://github.com/zitadel/zitadel_flutter/blob/main/lib/main.dart#L81-88
 ```
 
 What we'll change now, is the `_MyHomePageState` class to enable
-authentication via ZITADEL and remove the counter button of the starter application. We'll show the username of the authenticated user.
-
-We define the needed elements for our state:
-
-```dart
-var _busy = false;
-var _authenticated = false;
-var _username = '';
-final storage = const FlutterSecureStorage();
-```
+authentication via ZITADEL and remove the counter button of the starter application.
 
-Then the builder method, which does show the login button if you're not
-authenticated, a loading bar if the login process is going on and
-your name if you are authenticated:
+We'll show the username of the authenticated user.
 
 ```dart reference
-https://github.com/zitadel/zitadel_flutter/blob/main/lib/main.dart#L119-L159
+https://github.com/zitadel/zitadel_flutter/blob/main/lib/main.dart#L174-L197
 ```
 
 And finally the `_authenticate` method which calls the authorization endpoint,
 then fetches the user info and stores the tokens into the secure storage.
 
 ```dart reference
-https://github.com/zitadel/zitadel_flutter/blob/main/lib/main.dart#L45-L117
+https://github.com/zitadel/zitadel_flutter/blob/main/lib/main.dart#L115-L132
```

Comment:
I think copying the code is more sustainable than referencing it, which IMO outweighs the DRY principle. What do you think?